cloud/passportauth/src/user/BaseUser.ts

```diff
@@ -0,0 +1,32 @@
+export interface BaseUser {
```

Comment:
-1 for naming interfaces `Base*`, these should be base implementations designed to be extended and provide part of the functionality the interface requires